src/main/java/com/cloudlabs/server/compute/ComputeController.java

```diff
@@ -23,39 +23,21 @@ public class ComputeController {
 
 	// Create a new public instance with the provided "instanceName" value in the specified
 	// project and zone.
-	@PostMapping("/create")
-	public String create(@RequestBody JsonNode request)
+	
+	@DeleteMapping("/{project}/{zone}/{instanceName}")
```

Comment:
Is there a need to use DeleteMapping instead of Post or Get? 

The actual compute instance resource is not on the backend server. Our backend server is only responsible for routing the user's delete request to GCP's API.